### ISSUE TITLE
Premiere Pro and other apps look weird with dark theme

### ISSUE THREAD

**Issue by d0nutbag** [Rectify11/Installer#646]
### Description

So it was supposed to be a flat color, but the background is messed up and it now looks weird
![Captura de tela 2024-10-16 145828](https://github.com/user-attachments/assets/8fd066b2-129e-4d44-93bf-a29a47274e4e)

### Steps To Reproduce

Just installing Rectify11 made this issue, atleast for me

### Rectify11 Version

Version 3.2

### Windows Version

Windows 11 23H2 22631.4317

### Logs

N/A

**Comment by MuscularPuky** [comment]
Yes you should exclude some classes of Premiere Pro, or entire Premirere Pro process, in MiceForEveryone
and Mica Alt effect at Premiere Pro'll make more colorful and be cool
![스크린샷(642)](https://github.com/user-attachments/assets/14e541c9-b564-48f0-a596-79913a4ec6fa)
![스크린샷(687)](https://github.com/user-attachments/assets/fd49f566-e6e6-4530-afb1-790d5cde64c5)
